nettest/README.md

```diff
@@ -0,0 +1,26 @@
+## Network Test Suite for Demikernel
+
+This is repository contains hosts the network test suite for Demikernel.
```

Comment:
Could you elaborate on what you mean by `hosts` here?

Reply:
Fixed.